vision/product_search/product_search_list_products_in_product_set.rb

```diff
@@ -15,15 +15,14 @@
 # [START vision_product_search_list_products_in_product_set]
 require "google/cloud/vision"
 
-
 def product_search_list_products_in_product_set(
   project_id     = "your-project-id",
   location       = "us-west1",
   product_set_id = "your-product-set-id"
 )
-  client = Google::Cloud::Vision::ProductSearch.new
-
+  client           = Google::Cloud::Vision::ProductSearch.new
```

Comment:
FYI you didn't need to make this change to squish client instantiation and request setup together

```
require

def
  client =

  request setup
  request setup

  request execution

  response processing
  response processing
end

call method with args
```

^--- is totally fine

You can keep it as is. That was a lot of annoying `client          =` padding to stylize

Reply:
I went with newline after client. I agree it looked bad :D